### ISSUE TITLE
Can we get rid of releng?

### ISSUE THREAD

**Issue by SteveSchafer-Innovent** [eclipse-birt/birt#989]
In trying to track down where scripts are executed (both ant builds and other scripts), I see there are some scripts in build/org.eclipse.birt.releng.birtbuilder and build/org.eclipse.birt.releng.util that are not called anywhere.  I deleted those files and my local build still succeeds.   There is no pom.xml in birtbuilder or util.  There is a pom.xml in birtbuilder/extras but it isn't referenced as a module in any other pom so it may not be called either.  There are readme files in birtbuilder and util but they are very old (2007).  Has all this releng code been replaced by maven and can we delete it?

**Comment by wimjongman** [comment]
Does this build also still succeed if you do not have a local .m2 repo? Do the bundles get referenced in the build output?

**Comment by SteveSchafer-Innovent** [comment]
The local build still succeeds if I delete the local .m2 directory.

As for the other question:  Which bundles, and which build output are you referring to?   Neither build/org.eclipse.birt.releng.birtbuilder nor build/org.eclipse.birt.releng.util have any bundles (MANIFEST.MF files) nor any build output (target folders).

**Comment by wimjongman** [comment]
Sorry, I meant: do you see any reference to the scripts in the build output. If not I say go ahead and delete them. We can always pull them back from git.

**Comment by SteveSchafer-Innovent** [comment]
The util project doesn't seem to be referenced anywhere other than projectSet.psf so I deleted it.  Some files in the birtbuilder project are used as follows:

projectSet.psf
build/org.eclipse.birt.build/repoGen.properties:readmeFile=../org.eclipse.birt.releng.birtbuilder/extras/runtime_readme.txt
build/birt-packages/birt-wtp-integration-sdk/pom.xml: copy html files from extras
build/birt-packages/birt-sample-plugins/pom.xml: copy html files from extras
build/birt-packages/birt-runtime-osgi/build.xml: copy genReport.sh from extras
build/birt-packages/birt-runtime-osgi/build.xml: copy genReport.bat from extras
build/birt-packages/birt-runtime-osgi/build.xml: copy html files from extras
build/birt-packages/birt-runtime-osgi/build.xml: copy runtime_readme.txt from extras
build/birt-packages/birt-charts/ChartSDK/pom.xml: copy html files from extras
build/birt-packages/birt-charts/pom.xml: copy html files from extras
build/birt-packages/birt-charts/ChartRuntime/pom.xml: copy html files from extras
build/birt-packages/birt-report-framework/pom.xml: copy html files from extras
build/birt-packages/birt-tests-suite/pom.xml: copy files from eclipse/buildConfigs/birt.tests/test.scripts
build/birt-packages/birt-report-framework-sdk/pom.xml: copy html files from extras
viewer/org.eclipse.birt.report.viewer/Build_Web_Viewer_Instruction_For_BIRT2.2.html: mentioned several places

In birtbuilder I just deleted the .sh files and .xml files from the root directory.  Everything still builds locally so I'm creating a pull request.
